rhoas-cli-service-registry/README.adoc

```diff
@@ -449,9 +477,9 @@ $ rhoas service-registry artifact update --artifact-id=my-artifact my-artifact.j
 If you want to update the artifact from standard input,
 enter the following command:
 
-`$ rhoas service-registry artifact update --artifact-id=my_artifact`
+`rhoas service-registry artifact update --artifact-id=my_artifact`
```

Comment:
Line 449-452/477-480: If removing the OS prompt from the command, perhaps we should rephrase the first two lines of the Note as follows: “To update the artifact from standard input, use the <command> command.”

Reply:
Done